content/en/real_user_monitoring/reactnative/mobile_vitals.md

```diff
@@ -28,6 +28,8 @@ To access the RUM mobile app performance dashboard, scroll down to the Monitor P
 
 Understand your application's overall health and performance with the line graphs displaying metrics across various application versions. To filter on application version or see specific sessions and views, click on a graph.
 
+{{< img src="real_user_monitoring/react_native/rum_explorer_mobile_vitals-1.png" alt="Event Timings and Mobile Vitals in the RUM Explorer" style="width:90%;">}}
+
 You can also select a view in the RUM Explorer and observe recommended benchmark ranges that directly correlate to your application's user experience in the session. Click on a metric such as **Refresh Rate Average** and click **Search Views With Poor Performance** to apply a filter in your search query and examine additional views.
 
 ## Metrics
```

Comment:
can we add the `JS Rrefresh Rate` (or `JS Slow Renders`) in this list? @louiszawadzki or @fuzzybinary can advise if needed

Reply:
@louiszawadzki @fuzzybinary can you please help fill in the blanks here?